### PR TITLE
Task5_Module4

### PR DESCRIPTION
**Pull request by OlegMazurenko**
ConnectionString в appsettings.json добавил еще в первом задании.

### CODE REVIEW

File: DeliveryServiceEF/DeliveryServiceEF.Web/Filters/ProductExeptionFilter.cs [hunk 1]
```diff
@@ -10,16 +12,25 @@ namespace DeliveryServiceEF.Web.Filters
...
-        public ProductExeptionFilter(ILogger logger)
+        public ProductExeptionFilter(ILogger logger, IWebHostEnvironment env)
         {
             _logger = logger;
+            _env = env;
         }
 
         public void OnException(ExceptionContext context)
         {
+            if (_env.IsEnvironment("Production"))
```
Comment (yevhenbilushko) [inline DeliveryServiceEF/DeliveryServiceEF.Web/Filters/ProductExeptionFilter.cs]: _env.IsEnvironment("Production")
->
_env.IsProduction()

File: DeliveryServiceEF/DeliveryServiceEF.Web/Filters/ProductExeptionFilter.cs [hunk 2]
```diff
@@ -10,16 +12,25 @@ namespace DeliveryServiceEF.Web.Filters
...
+            {
+                _logger.LogInformation($"Error: {context.Exception.Message}");
```
Comment (yevhenbilushko) [inline DeliveryServiceEF/DeliveryServiceEF.Web/Filters/ProductExeptionFilter.cs]: LogInformation
->
LogError

**Review by yevhenbilushko — APPROVED** [review]
do u get connection string from appsettings.json?
merge and fix

**Comment by OlegMazurenko** [comment]
Yes, I'm getting connection string from appsettings.json and using it for DbContext in Startup

**Pull request merged**